applications/sasquatch/values-idfprod.yaml

```diff
@@ -0,0 +1,177 @@
+  connect:
+    enabled: false
+
+influxdb:
+  ingress:
+    enabled: true
+    hostname: data.lsst.cloud
+  resources:
+    requests:
+      # memory: 16Gi
+      # cpu: 2
+      memory: 8Gi
+      cpu: 1
+    limits:
+      # memory: 16Gi
+      # cpu: 2
+      memory: 8Gi
+      cpu: 1
+
+telegraf-kafka-consumer:
+  enabled: true
+  kafkaConsumers:
+    example:
+      enabled: true
+      replicaCount: 1
+      database: "lsst.example"
+      tags: |
+        [ "band", "instrument" ]
+      timestamp_format: "unix_ms"
+      timestamp_field: "timestamp"
+      topicRegexps: |
+        [ "lsst.example" ]
+#     backpack:
+#       enabled: true
+#       replicaCount: 1
+#       database: "lsst.backpack"
+#       timestamp_format: "unix"
+#       timestamp_field: "timestamp"
+#       topicRegexps: |
+#         [ "lsst.backpack" ]
+#     atcamera:
+#       enabled: true
+#       replicaCount: 1
+#       database: "lsst.ATCamera"
+#       timestamp_format: "unix_ms"
+#       timestamp_field: "timestamp"
+#       tags: |
+#         [ "Agent", "Aspic", "Location", "Raft", "Reb", "Sensor", "Source" ]
+#       topicRegexps: |
+#         [ "lsst.ATCamera" ]
+#     cccamera:
+#       enabled: true
+#       replicaCount: 1
+#       database: "lsst.CCCamera"
+#       timestamp_format: "unix_ms"
+#       timestamp_field: "timestamp"
+#       tags: |
+#         [ "Agent", "Aspic", "Cold", "Cryo", "Hardware", "Location", "Ps", "RTD", "Raft", "Reb", "Segment", "Sensor", "Source" ]
+#       topicRegexps: |
+#         [ "lsst.CCCamera" ]
+#     mtcamera:
+#       enabled: true
+#       replicaCount: 1
+#       database: "lsst.MTCamera"
+#       timestamp_format: "unix_ms"
+#       timestamp_field: "timestamp"
+#       tags: |
+#         [ "Agent", "Aspic", "Axis", "Canbus", "Cip", "Clamp", "Cold", "Controller", "Cryo", "Gateway", "Hardware", "Hip", "Hook", "Latch", "Location", "Ps", "RTD", "Raft", "Reb", "Segment", "Sensor", "Socket", "Source", "Truck" ]
+#       topicRegexps: |
+#         [ "lsst.MTCamera" ]
+
+kafdrop:
+  ingress:
+    enabled: true
+    hostname: data.lsst.cloud
+
+# rest-proxy:
+#   enabled: true
+#   ingress:
+#     enabled: true
+#     hostname: data.lsst.cloud
+#   kafka:
+#     topicPrefixes:
+#       - lsst.example
+#       - lsst.backpack
+#       - lsst.ATCamera
+#       - lsst.CCCamera
+#       - lsst.MTCamera
+
+chronograf:
+  ingress:
+    enabled: true
+    hostname: data.lsst.cloud
+
+  env:
+    GENERIC_NAME: "OIDC"
+    GENERIC_AUTH_URL: https://data.lsst.cloud/auth/openid/login
+    GENERIC_TOKEN_URL: https://data.lsst.cloud/auth/openid/token
+    USE_ID_TOKEN: 1
+    JWKS_URL: https://data.lsst.cloud/.well-known/jwks.json
+    GENERIC_API_URL: https://data.lsst.cloud/auth/openid/userinfo
+    GENERIC_SCOPES: openid
+    GENERIC_API_KEY: sub
+    PUBLIC_URL: https://data.lsst.cloud/
+    # STATUS_FEED_URL: https://raw.githubusercontent.com/lsst-sqre/rsp_broadcast/main/jsonfeeds/idfint.json
```

Comment:
Should I make an `idfprod.json` file in `rsp_broadcast`?

Reply:
Yes, that's the mechanism we use to populate the Chronograf news feed section, in the first page after you log into Chronograf. Can be used for notifications about this deployment.